### ISSUE TITLE
[Package Request]: flac

### ISSUE THREAD

**Issue by sidy3d** [microsoft/winget-pkgs#151382]
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://xiph.org/flac/download.html
Publisher: Xiph
Package Name: flac
Description: Free Lossless Audio Codec
Package Version: 1.4.3
Installer URL: 
https://github.com/xiph/flac/releases
or
https://ftp.osuosl.org/pub/xiph/releases/flac/

**Comment by Trenly** [comment]
[Policy] Zip binary

**Comment by fgimian** [comment]
Just an FYI that RareWares publish statically linked binaries of FLAC at https://www.rarewares.org/lossless.php

I'll submit a PR adding that version soon under a separate issue so we can keep this one open for tracking the official binaries.

**Comment by Trenly** [comment]
[Policy] Unblocked; Will require `ManifestVersion: 1.9.0` with `ArchiveBinariesDependOnPath: true`
